advanced/Templates/pihole-FTL.service

```diff
@@ -108,6 +114,9 @@ status() {
 
 ### main logic ###
 
+# catch sudden termination
+trap 'cleanup; exit 1' INT HUP TERM ABRT
```

Comment:
You can move the `exit 1` to the `cleanup` function.

Reply:
We can't, because we also use the cleanup function when we just stop the  service. See L99.